jujugui/static/gui/src/app/app.js

```diff
@@ -746,7 +746,9 @@ YUI.add('juju-gui', function(Y) {
       // nextProps on the lifecycle methods.
       ReactDOM.render(
         <window.juju.components.UserProfile
+          addNotification={this.db.notifications.add.bind(this)}
           currentModel={this.get('jujuEnvUUID')}
+          destroyModel={this.env.destroyModel.bind(this.env)}
```

Comment:
We're already passing `env` in below, is there a reason why you couldn't use that?